### PR TITLE
Form clone

### PR DESCRIPTION
**Pull request by rahalrazika**
Kindly review our project on  HTML forms.

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: style.css [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  color: var(--black);
+  font-weight: 300;
+  margin: 0.5rem;
+}
+
+#form-body {
+  padding: 1rem 2rem;
+}
+
+#terms {
```
Comment (andresporras3423) [inline style.css]: Don't overuse ID Selectors

File: style.css [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+.footer-links li a {
+  margin-right: 1rem;
+}
+
+.footer-links li a:hover {
+  text-decoration: underline;
+}
+
+
```
Comment (andresporras3423) [inline style.css]: Don't leave more than one empty line at the end of your CSS file

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle:
**Hello @rahalrazika and @Maryjanee** 
Good job so far with your first submission for this project :muscle: you have met most of the requirements for this project  :clap: :clap:
There are some changes to do before moving on:
- [ ] for the next code review please provide a more meaningful description of your changes
- [ ] the link of the screenshot seems to be broken
- [ ] Don't overuse ID selectors
- [ ] Check your CSS, make sure your that code passes all the linter checks
- [ ] check the appearance of the original page, some elements are not present
- [ ] It must show the Mint logo and text (logo on the left, text on the right)
- [ ] It must show the Quickbooks logo and text (logo on the left, text on the right)
- [ ] It shows the Turbotax logo and text (logo on the left, text on the right)

your page :point_down: :point_down: :point_down: :x: :x: :x:



The original :point_down: :point_down: :point_down: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:



your page :point_down: :point_down: :point_down: :x: :x: :x:



The original :point_down: :point_down: :point_down: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:



your page :point_down: :point_down: :point_down: :x: :x: :x:



The original :point_down: :point_down: :point_down: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:




Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon:  

Hi, @rahalrazika and @Maryjanee, great job :+1: You fixed all the comments pointed out by the previous TSE :100: Awesome !!! On this note, I hereby approved of your project :smile: However, before merging, kindly remove the button fixed at the footer
REN

After the fix, kindly merge to master and submit the project completion form
Good luck with your next projects :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**